githubapp/client_creator.go

```diff
@@ -156,6 +158,14 @@ func WithClientCaching(alwaysValidate bool, cache func() httpcache.Cache) Client
 	}
 }
 
+// WithClientTimeout sets the timeout on http.Client for all created clients. This will set a timeout
+// on all requests made by the client.
```

Comment:
style: `WithClientTimeout sets a request timeout for requests made by all created clients.`